terragrunt/aws/network/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "vpc_id" {
+  description = "VPC Id"
+  value       = module.url_shortener_vpc.vpc_id
+}
+
+output "private_subnet_ids" {
+  description = "List of the URL Shortner VPC private subnet ids"
+  value       = module.url_shortener_vpc.private_subnet_ids
+}
+
+output "api_id" {
```

Comment:
A super minor request - could you call this output something like `api_security_group_id` just so it's clearer that it's for an SG when we use it elsewhere.

Reply:
Done